index.html

```diff
@@ -82,14 +81,18 @@
           });
 
           //フォントの設定
-          styleSet.textContent = { ...styleSet.textContent,
-          fontFamily: '\'Comic Sans MS\', \'Arial\', sans-serif',
-          fontWeight: 'bold',
-          };
+          styleSet.textContent = Object.assign(
+            {},
+            styleSet.textContent,
+            {
+              fontFamily: '\'Comic Sans MS\', \'Arial\', sans-serif',
+              fontWeight: 'bold',
+            }
+          );
 
           window.WebChat.renderWebChat({
             directLine: window.WebChat.createDirectLine({ token: token }),
-            styleSet,
+            styleSet: styleSet,

```

Comment:
This shorthand is not supported by IE11 too, expanding it back to normal version.